src/npc.h

```diff
@@ -652,6 +652,10 @@ class npc : public player
 
 // Helper functions for ranged combat
  int  confident_range(int position = -1); // >= 50% chance to hit
+ /**
+  * Check if this NPC is blocking movement from the given position
+  */
+ bool is_blocking_position( const tripoint &p );
```

Comment:
An `is_*` function is usually const because it only checks a condition and that shouldn't affect the object at all.